tests/foreman/ui/test_gpgkey.py

```diff
@@ -33,7 +32,29 @@
 from robottelo.vm import VirtualMachine
 
 
-@ddt
+def get_random_gpgkey_name():
+    """Creates a random gpgkey name"""
+    return_value = generate_strings_list(remove_str='numeric', bug_id=1184480)
+    return return_value[randint(0, len(return_value)-1)]
+
+
+def invalid_gpgkey_names():
+    """List of invalid gpg key names for input testing.
+
+    Note: invalid_names_list() is overridden here as invalid_gpgkey_names
```

Comment:
how about:
```python
return generate_strings_list(len1=300, remove_str='numeric', bug_id=1184480)
```
? More common way and also it will include `numeric` whenever BZ is fixed.